src/locales/en-US.json

```diff
@@ -1255,6 +1255,7 @@
       "successNetworkPolicyUpdate": "Successfully updated %{policy}.",
       "successNextBootToast": "Applying changes to %{policy}. Changes made to %{policy} will take effect on next reboot."
     },
+    "acfUploadEnablementConfirmText": "Enabling this setting will allow an unauthorised user to upload ACF  for infinite period.\n You really confirm to enable this setting  ?",
```

Comment:
There are extra spaces in the text, Can those be removed?

Reply:
@Nikhil-Ashoka  Addressed